Doc/Makefile

```diff
@@ -213,8 +213,8 @@ dist:
 	rm dist/python-$(DISTVERSION)-docs-texinfo.tar
 
 check:
-	$(SPHINXLINT) -i tools -i $(VENVDIR) -i README.rst
-	$(SPHINXLINT) ../Misc/NEWS.d/next/
```

Comment:
Would you mind to add a comment to explain the purpose of the --severity option? Summarize what you wrote in the PR description ;-)

Reply:
Done.